source/configure-components-with-javascript/index.html.md.erb

```diff
@@ -0,0 +1,94 @@
+---
+title: Configure components with JavaScript
+weight: 76
+---
+
+# Configure components with JavaScript
+
+You can configure some of the components in GOV.UK Frontend to customise their behaviour or to [localise their JavaScript to use a language other than English](/localise-govuk-frontend/).
+
+You can configure a component by:
+
+- setting Nunjucks macro options
+- using data attributes in the HTML
+- passing a JavaScript object when creating an instance of a component
+- using the initAll function
+
+## Setting Nunjucks macro options
+
+If you're using the Nunjucks macros, you will find all of the configuration options for a component published on the GOV.UK Design System site.
+
+You can find the Nunjucks macro options by selecting the Nunjucks tab in the example box and selecting the Nunjucks macro options.
+
+## Passing configuration using data attributes in HTML
+
+If you're using HTML, you can pass configuration by adding data attributes to the component's outermost element (the element that has the `data-module` attribute). This is how our Nunjucks macros forward the configuration to the JavaScript components in the browser. Data attributes use kebab-case.
```

Comment:
```suggestion
If you’re using HTML, you can pass configuration by adding data attributes to the component’s outermost element (the element that has the `data-module` attribute). This is how our Nunjucks macros forward the configuration to the JavaScript components in the browser. Data attributes use kebab-case.
```
```suggestion
If you're using HTML, you can pass configuration by adding data attributes to the component's outermost element (the element that has the `data-module` attribute). This is how our Nunjucks macros forward the configuration to the JavaScript components in the browser. Data attributes use kebab-case.
```

Reply:
Middleman (or Kramdown, or something in between) takes care of making the quotes smart inside our content, so we can author them as `'` and they appear OK [on the generated page](https://deploy-preview-219--govuk-frontend-docs-preview.netlify.app/localise-govuk-frontend/).

Reply:
@romaricpascal But we're mixing styles? Straight single quote here, curly ones on lines 29, 45, 49 etc?

That alright?

Reply:
Oh, I see. Seems the other files are quite lenient on having those consistent, but I'll tidy it up in these new pieces of documentation. Wonder if that's something we can lint, maybe with `remark-lint` :D